renn/data/tokenizers.py

```diff
@@ -1,11 +1,67 @@
 """Text processing."""
 
-import tensorflow_datasets as tfds
-import tensorflow_text as tft
+from collections import Counter
+import itertools
 
+from renn.data import wordpiece_tokenizer_learner_lib as learner
```

Comment:
Consider renaming "learner" to something more specific like "vocab_learner"

Reply:
Done